### PR TITLE
adds beheading

### PR DESCRIPTION
**Pull request by sylvia-from-fulp-station**
## About The Pull Request

yeah.......
adds one line to the code for heads allowing them to be dismembered like other limbs.

tested: 
-cranial fissures still exist and happen a few hits before beheading.
-the plasma cutter does not dismember the head at any amount of damage.

## Why It's Good For The Game

it's a scary thing that antagonists or shit sec can do which provides a unique situation for medical to deal with.

a beheading is loud, messy and creates more evidence. cremation / gibbing is still faster rewarding well-planned murder.

there is a cool interaction with lings being able to survive without a head.

## Changelog
:cl:
balance: many melee hits to the head will dismember it.
/:cl:

### CODE REVIEW

**Comment by breadcrombs** [comment]
https://github.com/tgstation/tgstation/pull/83479 revert this for plasma cutter decap

File: wallstation_modules/revertbeheading/code/head.dm [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Editing: code/modules/surgery/bodyparts/head.dm
+/obj/item/bodypart/head
+    // allows dismembering heads
+	can_dismember = TRUE
```
Comment (SmArtKar) [inline wallstation_modules/revertbeheading/code/head.dm]: Inconsistent embeds are bad, if linters weren't down they'd probably error here

Comment (sylvia-from-fulp-station) [reply wallstation_modules/revertbeheading/code/head.dm]: no clue why it shows up like this on github, in editor it's embedded once on each line, i removed the comment as its just holdover from the massmeta revert i copied this from

**Comment by grungussuss** [comment]
> https://github.com/tgstation/tgstation/pull/83479 revert this for plasma cutter decap

Reverting that will allow stuff like pistols and laser guns to dismember heads even if dismemberment was disabled. Since with this change it will be enabled anyways, it doesn't matter

File: wallstation_modules/revertbeheading/code/head.dm [hunk 2]
```diff
@@ -0,0 +1,2 @@
+/obj/item/bodypart/head
+	can_dismember = TRUE
```
Comment (Artemchik542) [inline wallstation_modules/revertbeheading/code/head.dm:3]:
```suggestion
// ORIGINAL FILE: code/modules/surgery/bodyparts/head.dm
/obj/item/bodypart/head
	can_dismember = TRUE
```

Is this repository supposed to contain some documentation for the modules? Like readme.md or just comments.

Comment (sylvia-from-fulp-station) [reply wallstation_modules/revertbeheading/code/head.dm:3]: I got no idea. For something this simple that isn't editing any TG files i think a comment is fine.

Comment (grungussuss) [reply wallstation_modules/revertbeheading/code/head.dm:3]: it's not required as we can just search for the typepath, useful nonetheless so keep it

**Review by grungussuss — APPROVED** [review]
if it becomes a balance problem we can always remove it again, this can be a fun experiment.

**Pull request merged**